lib/rubygems/commands/install_command_extend.rb

```diff
@@ -34,8 +34,6 @@
 
         installer.say "Verifying #{gem_path}"
 
-        raise Gem::CommandLineError, "#{gem_path} is not a file" unless File.file?(gem_path)
```

Comment:
From what @jchestershopify and I could tell, when these pre-install hooks are called, Rubygems has already validated that the given package is a valid gem, both locally and remotely. If the file does not exist or is not a valid gemfile, no package exists on the installer at line 32. Plus, Rubygems raises an error:
```
gem install --verify-signatures blahh

[snip of irrelevant warning]
         
ERROR:  Could not find a valid gem 'blahh' (>= 0) in any repository
ERROR:  Possible alternatives: blah
```

We're confident this is unreachable, but if there's something we've overlooked please let us know.